examples/mobile/UIComponents/components/dialogs/dialog.html

```diff
@@ -456,6 +461,27 @@ <h1>
 					<button class="ui-btn" data-style="flat">Action</button>
 				</div>
 			</div>
+			<div class="ui-popup" id="slider-dialog">
+				<div class="ui-popup-header">
+					Dialog header
+				</div>
+				<div class="ui-popup-content">
+					<div class="ui-popup-slider-text">
+						<div class="text-left">Left</div>
+						<div class="text-right">Right</div>
+					</div>
+					<div class="ui-popup-slider">
+						<div class="container">
+							<input max="100" min="0" type="range" value="50" />
+						</div>
+					</div>
```

Comment:
Is there any component for the ui-slider?
I think it would be good to define the slider without ui-popup-* like as,
``` html
<div class="ui-popup-content">
  <input class="ui-slider" data-left-label="Left" data-right-label="Right" max="100" min="0" type="range" value="50" />
</div>
```